apps/admin-gui/src/app/shared/components/delete-owner-dialog/delete-owner-dialog.component.ts

```diff
@@ -35,19 +35,16 @@ export class DeleteOwnerDialogComponent implements OnInit {
 
   onSubmit(): void {
     this.loading = true;
-    if (this.owners.length) {
-      this.ownersManagerService.deleteOwner(this.owners.pop().id).subscribe(
-        () => {
-          this.onSubmit();
-        },
-        () => (this.loading = false)
-      );
-    } else {
-      this.translate.get('DIALOGS.DELETE_OWNER.SUCCESS').subscribe((successMessage: string) => {
-        this.loading = false;
-        this.notificator.showSuccess(successMessage);
-        this.dialogRef.close(true);
-      });
-    }
+    const ownerIds = this.owners.map((owner) => owner.id);
+    this.ownersManagerService.deleteOwners(ownerIds).subscribe({
+      next: () => {
+        this.translate.get('DIALOGS.DELETE_OWNER.SUCCESS').subscribe((successMessage: string) => {
+          this.loading = false;
+          this.notificator.showSuccess(successMessage);
+          this.dialogRef.close(true);
+        });
```

Comment:
I have one comment regarding the translations as well. Could you please here and other affected files use the `instant` translation instead of `get` ? It is a little bit cleaner that way.